torch_xla/distributed/xla_multiprocessing.py

```diff
@@ -55,7 +55,7 @@ def _create_gpu_devices(num_gpus):
 
 
 def _parse_workers_config(config):
-  # XRT_WORKERS='worker:0;ismz9:25822'
+  # PJRT_DEVICE=TPU
```

Comment:
a lot of functions in this file is only used by XRT, we should do a clean up afterward.

Reply:
@will-cromar to review change in this file

Reply:
sure, will do and saved in the list of full list of OpenXLA miration list

Reply:
has revert this change and @will-cromar will have follow-up PR for XRT contain this change